test/runtests.jl

```diff
@@ -12,6 +13,12 @@ else
            * " (but not the test summary).")
 end
 
+# minimize messages when testing locally, all messages on CI
+test_logger = if tryparse(Bool, get(ENV, "CI", "false")) === true
+    current_logger()
+else
+    NullLogger()
+end
```

Comment:
I am not sure we want do do this.

Reply:
Let me explain. When I jumped into the `Referencetests` suite, and run `] test ReferenceTests` before editing files in my fork, the messages generated by `@test_reference` were clearly confusing (they tend to indicate a failure where there is none, e.g. when testing using `@test_throws`). IMO, it is better if only the test summary appear locally (all messages are kept in CI).

Reply:
Moved to https://github.com/JuliaTesting/ReferenceTests.jl/pull/118.